File: Serene3/Serene3.Web/Modules/ServerTypes/com/reviewColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { reviewRow } from "./reviewRow";

export interface reviewColumns {
    Id: Column<reviewRow>;
    Name: Column<reviewRow>;
    Email: Column<reviewRow>;
    Subject: Column<reviewRow>;
    Description: Column<reviewRow>;
}

export class reviewColumns extends ColumnsBase<reviewRow> {
    static readonly columnsKey = 'com.review';
    static readonly Fields = fieldsProxy<reviewColumns>();
}

File: Serene3/Serene3.Web/Modules/com/review/reviewColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace Serene3.com.Columns;

[ColumnsScript("com.review")]
[BasedOnRow(typeof(reviewRow), CheckNames = true)]
public class reviewColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    [EditLink]
    public string Name { get; set; }
    public string Email { get; set; }
    public string Subject { get; set; }
    public string Description { get; set; }
}